exercise_10.py

```diff
@@ -0,0 +1,13 @@
+import random
+
+a = [1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89]
+b = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13]
+
+print(set([x for x in a if x in b]))
+
+print("Random lists")
+a = random.sample(range(100), random.randint(20,40))
+b = random.sample(range(100), random.randint(20,40))
```

Comment:
Sprytnie bo `random.sample` zwraca elementy unikalne :)

Zmień to tak by losowe listy mogły zawierać powtarzające się elementy.